src/naive/internals.rs

```diff
@@ -368,33 +370,37 @@ impl fmt::Debug for Of {
 /// (month, day of month and leap flag),
 /// which is an index to the `MDL_TO_OL` lookup table.
 #[derive(PartialEq, PartialOrd, Copy, Clone)]
-pub(super) struct Mdf(pub(super) u32);
```

Comment:
The value didn't need to be public. Making it private makes it clearer the value is always correct, and thus in bounds for array indexing.